layouts/shortcodes/feature-gate-description.html

```diff
@@ -0,0 +1,18 @@
+{{- $id := .Get "name" -}}
+{{- $featureGatesBundle := site.GetPage "page" "docs/reference/command-line-tools-reference/feature-gates" -}}
+{{- $featureGates :=  $featureGatesBundle.Resources.ByType "page" -}}
```

Comment:
`$featureGates` custom variable appears to be unnecessary since its value is not utilized anywhere.